### PR TITLE
1pw import adds

### PR DESCRIPTION
**Pull request by fschillingeriv**

### CODE REVIEW

File: _articles/importing/import-from-1password.md [hunk 1]
```diff
@@ -7,47 +7,61 @@ popular: false
...
 
-Importing your data from 1Password into Bitwarden is easy. 1Password has two versions of their application: 1Password 4 and 1Password 6 and 7. Depending on which operating system and version of 1Password and you are using, follow the proper steps below.
+To migrate from 1Password to Bitwarden, export your data from 1Password and use the Bitwarden Import Tool to quickly fill your Vault with logins.
 
-## Export Your 1Password 1pif Logins
+Export your data from 1Password as a `.1pif` or `.csv`, depending on which version and operating system you are using:
 
-{% callout info %}
-This set of instructions is only for:
+- `.1pif` files can be exported by **1Password 4 on Windows**, **1Password 6 on macOS**, and **1Password 7 on macOS** (*latest*).
```
Comment (MGibson1) [inline _articles/importing/import-from-1password.md:14]: windows is capable of exporting 1pif.

I'm also not clear on what the (*latest*) tag is buying us here. I don't know how far back the 1pif format has been supported. Is it trying to say that 1password 7 is the latest version?

File: _articles/importing/import-from-1password.md [hunk 2]
```diff
@@ -7,47 +7,61 @@ popular: false
...
+- `.csv` files can be exported by **1Password 6 on Windows or macOS** and **1Password 7 on Windows or macOS** (*latest*).
 
-- 1Password 4 users on macOS and Windows (only the .agilekeychain format is supported for Windows users)
-- 1Password 6 and 7 users on macOS
+## Export as .1pif
 
-These instructions may vary slightly for macOS users.
-{% endcallout %}
+Complete the following steps to export your 1Password data as a `.1pif` from **1Password 4 on Windows**, **1Password 6 on macOS**, or **1Password 7 on macOS**:
```
Comment (MGibson1) [inline _articles/importing/import-from-1password.md:19]: Do we have reason to think the 1pif format has evolved and we only support these versions? I haven't done any testing with it

File: _articles/importing/import-from-1password.md [hunk 3]
```diff
@@ -7,47 +7,61 @@ popular: false
...
-6. Select a folder to save your export file to (recommended to use your desktop folder). Click save to export your .1pif data file.
+1. In the 1Password Desktop App, navigate to the vault you'd like to export.
 
-## Export your 1Password 6 &amp; 7 csv logins from Windows
+   1Password does not allow you to export from multiple vaults at once.
+2. Select **File** &rarr; **Export**.
 
-{% callout info %}
-This set of instructions is only for:
+   *On macOS, you will also need to select **All Items..**.*
```
Comment (MGibson1) [inline _articles/importing/import-from-1password.md:26]: **File** &rarr; **Export** &rarr; **All Items...** would be clearer to me

**Review by MGibson1 — COMMENTED** [review]
Looks good to me, I just have a few questions related to compatibility.  I get that it was just copied over, but since we're updating documentation it feels like a good time to re-test compatibity as well.

File: _articles/importing/import-from-1password.md (hunk 1 above)
Comment (fschillingeriv) [reply _articles/importing/import-from-1password.md:14]: Windows running v7? I based this list on it not being an option for 1PW Windows here: https://support.1password.com/export/. Admittedly, I don't have a Windows machine to test it on.

Comment (MGibson1) [reply _articles/importing/import-from-1password.md]: yup, don't know why 1pass's documentation is wrong, but here's my windows desktop version 7.6.785 offering to save 1pif

image

**Comment by fschillingeriv** [comment]
@MGibson1 I redid this article to (hopefully) make it aggressively simpler and accurate across all version/OSs. Also pulled out some of the claims about what-works-with-what in order to prevent us from having to re-validate that periodically. LMK what you think.

**Comment by MGibson1** [comment]
Looks good to me!

**Review by tgreer-bw — APPROVED** [review]

**Review by MGibson1 — APPROVED** [review]

**Pull request merged**